<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckCycleTimeForm</class>
 <widget class="GUIFormBase" name="CheckCycleTimeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check CycleTime</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSettingTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxCurrentCycleTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>SettingTime</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>CurrentCycle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
   <property name="icon">
    <iconset resource="CheckCycleTime.qrc">
     <normaloff>:/Qtres/Setting.png</normaloff>:/Qtres/Setting.png</iconset>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSettingError">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Error time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CheckCycleTime.qrc"/>
 </resources>
 <connections/>
</ui>
